pom.xml

```diff
@@ -22,6 +22,7 @@
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
     <node.version>6.4.0</node.version>
     <npm.version>3.10.3</npm.version>
+    <yarn.version>0.23.0</yarn.version>
```

Comment:
also people tend to use it when they are checking they have a new enough version for doing things outside of mvn (but yeah, redundant technically)